src/Api/Projects.php

```diff
@@ -1188,6 +1188,50 @@ public function addProtectedBranch($project_id, array $parameters = [])
         return $this->post($this->getProjectPath($project_id, 'protected_branches'), $parameters);
     }
 
+    /**
+     * @param int|string $project_id
+     * @param string     $branch_name
+     *
+     * @return mixed
+     */
+    public function removeProtectedBranch($project_id, string $branch_name)
+    {
+        return $this->delete($this->getProjectPath($project_id, 'protected_branches/'.self::encodePath($branch_name)));
+    }
+
+    /**
+     * @param int|string $project_id
+     * @param array      $parameters
```

Comment:
I cannot merge this with arbitrary undocumented parameters. Please either delete the parameter, or document those that are permitted, enforced with the options resolver.

Reply:
I copied how the `protectedBranches` method was implemented. Is there some extra documentation somewhere that I missed, that I should add to?